app/controllers/billing.js

```diff
@@ -0,0 +1,6 @@
+import Controller from '@ember/controller';
```

Comment:
Is it possible to remove this entire file?

Reply:
@allouis Don't think so, as we'll need atleast the basic skeleton controller here with `guid` I think😬